MyAssignment2.Rmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "MyAssignment2"
+author: "Elliot Hu-Au"
+date: "October 25, 2018"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+R Markdown for HUDK 4050: Assignment 2
+
+Part II
+```{r}
+
+
+E1<-dplyr::select(EDGE,count)
+student_count<- EDGE %>% group_by(EDGE$to) %>% summarize(sum = sum(count))
+names(student_count)[1]<-"id"
+new_count<- merge(VERTEX,student_count,by="id")
+
+g <- graph.data.frame(EDGE, directed=TRUE, vertices=all_totals)
+
+teach_count<- EDGE %>% group_by(from) %>% summarize(sum(count))
+names(teach_count)[2]<-"sum"
+names(teach_count)[1]<-"id"
+new_teach_count<- merge (VERTEX,teach_count, by="id")
+
+all_totals<-dplyr::bind_rows(new_teach_count,new_count)
+nodeSize<- setNames(c(all_totals$sum),c(all_totals$id))
+
+
+plot(g,layout=layout.fruchterman.reingold, vertex.color=all_totals$gender, edge.width=EDGE$count, vertex.size=all_totals$sum*3)
```

Comment:
Sometimes it requires you to feed igraph the color codes for the vertices, so you would have to code your gender as color names or hexidecimal colors, eg. male = pink, female = blue.